e2e-test/tsps_e2e_test.py

```diff
@@ -231,7 +280,9 @@ def download_with_azcopy(sas_url):
     for key, value in pipeline_output.items():
         logging.info(f"attempting to retrieve {key} output")
         download_with_azcopy(value)
```

Comment:
i know these are basically empty but should we clean up the downloaded file and the file we download?  At the very least we could put both of those files in temp space so they get cleaned up automatically